### ISSUE TITLE
React Video Player - Cannot interact with some elements using Voiceover (Video)

### ISSUE THREAD

**Issue by naganjaneya17** [cookpete/react-player#1924]
We are using React player and facing accessibility below issues.  How to rectify focus contrast and voiceover issues while using VDI in safari browser.
- Keyboard focus contrast for video controls is failing in Safari 
- Cannot interact with some elements using Voiceover in Safari
